adminpage/api/crud/crud_training.py

```diff
@@ -85,18 +86,34 @@ def get_trainings_for_student(student: Student, start: datetime, end: datetime):
                        'g.id AS group_id, '
                        'g.name AS group_name, '
                        'tc.name AS training_class, '
-                       'COALESCE(a.hours, 0) AS hours, '
                        'FALSE AS can_grade '
                        'FROM enroll e, "group" g, sport s, training t '
-                       'LEFT JOIN attendance a ON a.student_id = %s AND a.training_id = t.id '
                        'LEFT JOIN training_class tc ON t.training_class_id = tc.id '
-                       'WHERE t.start > %s AND t."end" < %s '
+                       'WHERE t.start > %(start)s AND t."end" < %(end)s '
```

Comment:
Если ивент начался на прошлой неделе или закончился на следующей, то он не отобразится в календаре.

Фикс:
```suggestion
                       'WHERE (t.start > %(start)s OR t."end" < %(end)s) '
```

Reply:
Такой фикс не сработает, если ивент идет > 1 недели. Также тут выделятся лишние события.
Я вроде обновил на рабочий вариант.